docs/demo/gaiden-css/components/media/default/demo.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="/gaiden-css/gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+</head>
+
+<body>
+  <div class="container">
+    <div class="media col-small-12 col-medium-4 col-normal-4">
+      <div class="media__image thumbnail space-box-medium">
```

Comment:
`<dl>` is a _description list_, which has a semantic value for a group of terms. This is not a case to use `<dl>`.

Bellow is an example of `<dl>` use: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl